media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe

```diff
@@ -71,7 +79,7 @@ TEST()
 {
 	cd Test
 	runConfigure ./configure
-# fails to configure due to missing sdl_gfx
-	make
+# fails to configure due to missing sdl2_gfx
```

Comment:
Maybe inspecting the configure log could make this work.

Reply:
Already checked config.log, no real information there